README.adoc

```diff
@@ -110,3 +108,25 @@ dependencies {
     implementation "care.data4life.hc-util-sdk-kmp:util-android:$version"
 }
 ----
+
+== Roadmap
+
+This project is work in progress. We are working on adding more functionality, guidelines, documentation and other improvements.
+
+== Contributing
+
+You want to help or share a proposal? You have a specific problem?
+
+* Open an issue
+
+=== Contribution Guide
+
+TODO will be added soon.
```

Comment:
Maybe without `TODO`?

Reply:
I left TODO as it is easily searchable.

Reply:
Ok, so we should provide a follow up soonish.